slotfills/sections/checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { PanelRow, CheckboxControl } from '@wordpress/components';
+import Field from '@/interfaces/field';
+
+type Props = {
+  field: Field,
+  valueHook: Function,
+  index?: Number,
+  label?: string,
+};
+
+export default function Checkbox({
+  field: {
+    name,
+    checked_value: checkedValue,
```

Comment:
Following up on my comment on the other PR - if `checked_value` is coming from somewhere like an API, this is the correct way to transform that data into a local variable that satisfies ESLint, so 👍